.circleci/config.yml

```
@@ -1,80 +1,151 @@
----
-version: 2
+version: 2.1
```

Nitpick: The `---` is a "document start marker". It's usually good practice to include a `tag` showing what version of the YAML spec that is used. like this:

```suggestion
%YAML 1.2
---

version: 2.1
```